recipes/Media/ASR/seq2seq/hparams/train.yaml

```diff
@@ -0,0 +1,149 @@
+# ################################
+# Model: CRDNN + DNN + CTC + Softmax
+# Authors modifier: Gaelle Laperriere
+# ################################
+
+# ------ Paths and parameters
+
+# Seed needs to be set at top of yaml, before objects with parameters are made.
+seed: 4242
+__set_seed: !!python/object/apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/media_ASR/<seed>
+cer_file_test: !ref <output_folder>/cer_test.txt
+ctc_file_test: !ref <output_folder>/ctc_test.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files:
+data_folder: !PLACEHOLDER
+csv_train: !ref <data_folder>/train.csv
+csv_valid: !ref <data_folder>/dev.csv
+csv_test: !ref <data_folder>/test.csv
+
+# Data parameters:
+# With data_parallel batch_size is split into N jobs.
+# With DDP batch_size is multiplied by N jobs.
+batch_size: 4
+test_batch_size: 2
+# We remove utterances longer than 90s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 90.0
```

Comment:
What is the longest sentence in this dataset? 90s looks huuuuuuge

Reply:
I don't remember exactly. Most are 20s maximum. We decided to let 90 for one or two exceptions that where bigger than 90s. It has been verified.